Packs/MicrosoftGraphFiles/Integrations/MicrosoftGraphFiles/MicrosoftGraphFiles.yml

```diff
@@ -537,8 +537,11 @@ script:
     - contextPath: MsGraphFiles.ListChildren.Children.OdataContext
       description: The OData query.
       type: String
+    - contextPath: MsGraphFiles.ListChildren.NextToken
+      description: The token for the next page.
+      type: String
   - arguments:
-    - description: Keyword used to search for sites across a SharePoint tenant. If a keyword is not provided, it returns all sites)
+    - description: Keyword used to search for sites across a SharePoint tenant. If a keyword is not provided, it returns all sites.
```

Comment:
Do we want to add to the description that its only relevant for the full sites' permissions?

Reply:
Done